compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -359,6 +359,9 @@ error.invalid.index.expr.struct.field.access=\
 error.invalid.index.expr.tuple.field.access=\
   invalid index expression: expected integer literal
 
+error.invalid.finite.type.index.expr=\
+  invalid index expression: invalid finite type value space ''{0}''
```

Comment:
In the spec, we don't use the word "finite type", This should give an error message, which indicates, given value is doesn't belong to tuple size.